handlers/_swagger.js

```diff
@@ -419,40 +423,52 @@ function validateRequestParameters(req, data, logger, callback) {
 
         } else if (parm.in === 'body') {
             var result = swaggerUtil.validateJSONType(parm.schema, req.body);
-
-            if (!result.valid) {
+            var polymorphicValidationErrors = [];
```

Comment:
ok, so that's alright.
I was thinking to "fail fast" by returning any error from the default validation immediately.